graphviz/plan.sh

```diff
@@ -24,12 +24,10 @@ pkg_build_deps=(
     core/pkg-config
 )
 pkg_lib_dirs=(lib)
+pkg_bin_dirs=(bin)
 pkg_include_dirs=(include)
 
 do_prepare() {
-  # TODO: if we don't copy this aclocal will fail. need to figure out how to fix this
```

Comment:
Is this no longer true?

Reply:
@eeyun Not from my testing. Due to usage of `core/pkg-config` it finds what it needs.